resource-timing/embed-navigate-back.html

```diff
@@ -0,0 +1,16 @@
+<!DOCTYPE html>
+<html>
+<head>
+<meta charset="utf-8" />
+<title>Resource Timing embed navigate - back button navigation</title>
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+<script>
+    async_test(t => {
+        window.open("resources/embed-navigate-back.html");
```

Comment:
Based on running the tests locally, it'd help a lot to add something like
```js
const openee = window.open(...);
t.add_cleanup(() => openee.close());
```
here and elsewhere where this pattern is used.

Reply:
Nice! Thanks :)